### ISSUE TITLE
auto start permissions

### ISSUE THREAD

**Issue by zhaba17** [stanfieldr/ghetto-skype#135]
Hi, when I try to configure ghetto-skype to open on login to the desktop I get a message about how it does not have permission to write to /home/user/.config/autostart .... 

How do I solve this? Or edit a configuration file ... 

Thanks

**Comment by stanfieldr** [comment]
If you do this command, it should fix the problem.
```bash
chmod u+rw /home/user/autostart
```

**Comment by zhaba17** [comment]
sorry. I replied to this by email, not realising it wouldn't show up in the thread. So I'll post what I wrote here...

and the current permisions are (8/11/17),

-rw-rw-rw- 1 simon simon 223 дек 18  2016 Ghetto Skype.desktop

"Thanks for the response. I tried that and no change. The message is,

Ghetto Skype does not have permission to write to /home/simon/.config/autostart/ghetto-skype.desktop

so I tried to change permissions on the file, but can't as is symlink. So I then tried going to

/usr/share/applications

and

sudo chmod 666 "Ghetto Skype.desktop"

the permissions are now

-rw-rw-rw- 1 root root 223 дек 18  2016 Ghetto Skype.desktop

I am using Maui Linux, which is the latest KDE Plasma desktop built on Ubuntu 16.04. 

Many thanks"
